.gitignore

```diff
@@ -100,6 +100,9 @@ build/Release
 node_modules/
 jspm_packages/
 
+# TypeScript v1 declaration files
+typings/
```

Comment:
Is this directory auto-generated?

Reply:
No, I just renamed the `typings` directories to `types` .
And that ignore code was previously removed but now re-added just for the sake of v1 declaration files.

Reply:
Ok, thanks for the clarification. We use TypeScript 4. Do we still need the directory? Asking cause I don't know what is the issue with TypeScript v1 typings.

Reply:
We don't necessarily need it but I kept it much like other directories that don't exist in out project,
but are still best to ignore them such as `jspm_packages`, `bower_components`, `lib-cov` for jscoverage, and `.nyc_output` for nyx testing.
We don't have them on the project right now, but it's in-case of the future, or someone pull request a file they don't mean.
We could also resolve in removing all files not pertaining to the project and gradually add them back as time goes on.

Reply:
Yea, totally makes sense.